### PR TITLE
DS: Build (Basic) Table

### PR DESCRIPTION
**Pull request by nearbycoder**
This PR introduces Tables
[DS: Build Table (Basic)](https://app.clubhouse.io/firehydrant/story/14042/ds-build-table-basic)

### CODE REVIEW

File: lib/components/table/Table.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Table as ChakraTable, Td as ChakraTd } from '@chakra-ui/react';
+
+export function Table({ children, ...rest }) {
+  return <ChakraTable {...rest}>{children}</ChakraTable>;
+}
+
+export function Td({ children, isInteractive, ...rest }) {
+  return (
+    <ChakraTd {...rest} py={isInteractive ? 'mg2' : '20px'}>
```
Comment (nearbycoder) [inline lib/components/table/Table.jsx:10]: isInteractive prop is used to remove the `2x mg2` spacing for when you have a button in the Td. Not set on the name if you have another one in mind @caseymhunt.

File: lib/components/table/Table.stories.js [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+            <IconButton variant="secondary" name="search" />
+          </ButtonGroup>
+        </Td>
+      </Tr>
+    </Tbody>
+  </Table>
+);
+
+export const RowAlignment = () => (
+  <Table isWrapping>
```
Comment (nearbycoder) [inline lib/components/table/Table.stories.js:329]: @caseymhunt all this does is tell the text to align to the top when there are multiple lines. Unless you have another CSS option that doesn't require Javascript I don't think their is a decent way for us to do this automatically.

**Review by nearbycoder — COMMENTED** [review]
@caseymhunt added my thoughts

File: lib/components/table/Table.jsx (hunk 1 above)
Comment (caseymhunt) [reply lib/components/table/Table.jsx:10]: 🤔  my only other idea would be _even more literal_ like `hasButtons`

File: lib/components/table/Table.stories.js (hunk 2 above)
Comment (caseymhunt) [reply lib/components/table/Table.stories.js:329]: So I'm playing with this and regardless if the prop is included or not, I'm not seeing any changes. What am I missing? How do I see what this does?

File: lib/components/table/Table.jsx (hunk 1 above)
Comment (caseymhunt) [reply lib/components/table/Table.jsx:10]: is this still necessary? If so, we should probably showcase it in one of the stories.

Comment (nearbycoder) [reply lib/components/table/Table.jsx:10]: Yes it's necessary and is showcased in a few of the stories with buttons

**Review by caseymhunt — APPROVED** [review]
🚢  it

**Review by caseymhunt — APPROVED** [review]

**Pull request merged**